static/css/things.css

```
@@ -11,32 +11,25 @@
 
 #things-container {
   position: relative;
-  top: 50%;
-  transform: translateY(-50%);
-  max-height: 100%;
-  display: flex;
-  flex-direction: column;
+  margin: auto;
+  width: calc(100% - 10rem);
```

This breaks the vertical alignment of Things, and the "Click + to add Thing" hint in the empty state.

You could use flexbox for the vertical aligment instead of the old transform approach:

```suggestion
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
```